modules/installation-obtaining-installer.adoc

```diff
@@ -99,14 +90,9 @@ ifndef::ibm-z,ibm-z-kvm,private,vsphere[* You have a computer that runs Linux or
 .Procedure
 
 ifndef::openshift-origin[]
-. Access the link:https://console.redhat.com/openshift/install[Infrastructure Provider] page on the {cluster-manager} site. If you have a Red Hat account, log in with your credentials. If you do not, create an account.
-ifndef::ash[]
-. Select your infrastructure provider.
-endif::ash[]
-ifdef::ash[]
-. Select *Azure* as the cloud provider.
-endif::ash[]
-. Navigate to the page for your installation type, download the installation program that corresponds with your host operating system and architecture, and place the file in the directory where you will store the installation configuration files.
+. Navigate to the link:https://access.redhat.com/downloads/content/290/[OpenShift Container Platform downloads page] on the Red{nbsp}Hat Customer Portal.
```

Comment:
@skopacz1 

it looks like this link does not work for me:
```
Active Subscription Required
You will need an active subscription before you can download this software. [Learn more about how a Red Hat subscription works](https://access.redhat.com/subscription-value/).
```

But the original link (https://console.redhat.com/openshift/install) works: I could download the installation binary by using the same account without _subscription_

Is it expected?

Reply:
I can access the link and downloads with my RH associate account, are you using your associate account or a test account that doesn't have permissions related to being an associate/subscriber?

I am guessing that the downloads page of the portal website intentionally limits access for people without a subscription, but I'm not sure.

If the account you used is supposed to have those subscriber permissions, then I think that is an unexpected issue.

Reply:
> I can access the link and downloads with my RH associate account, are you using your associate account or a test account that doesn't have permissions related to being an associate/subscriber?
> 
> I am guessing that the downloads page of the portal website intentionally limits access for people without a subscription, but I'm not sure.

That's the issue: after this PR, only the valid subscriber can download the client. 
I thought it was supposed to be downloadable by anyone with an external RH account.